basic_integral/basic_integral.cpp

```diff
@@ -0,0 +1,25 @@
+#include <iostream>
+#include <cmath>
+using namespace std;
+
+int main() {
+	int N = 10, b, i;
+	double sum, fx;
+	double p = 2*acos(0.0);
+	
+	cout << "How many iterations?" ;
+	cin >> b ;
+	
+	N = 10 * pow(b-1,2);

```

Comment:
You had already initialized N to 10. You're repeating the value 10 here. Meaning the initialization was redundant.